x/datapool/keeper/pool.go

```diff
@@ -217,6 +115,12 @@ func (k Keeper) CreatePool(ctx sdk.Context, curator sdk.AccAddress, deposit sdk.
 	return newPool.GetPoolId(), nil
 }
 
+func (k Keeper) isRegisteredOracle(ctx sdk.Context, oracleAddress sdk.AccAddress) bool {
+	store := ctx.KVStore(k.storeKey)
```

Comment:
When checking the oracle is registered or not, we have to use `x/oracle` keeper, not `x/datapool`. Because we store the info of oracle in `x/oracle`. So let's import the `x/oracle` module into the `x/datapool` keeper and use it. 

For that, you may have to add `x/oracle` keeper in `x/datapool` keeper struct (`x/datapool/keeper/keeper.go`).
And then, you can check it by `k.oraclekeeper.isRegisteredOracle()`.

Reply:
Without this comment, I resolved other comments in [658d72e](https://github.com/medibloc/panacea-core/pull/341/commits/658d72e32e99079febba08d26e022865fc500408).

Reply:
You can check this discussion in [bebca0b](https://github.com/medibloc/panacea-core/pull/341/commits/bebca0b48d2680429c31a5c2dbc1eb28915309d0).

Reply:
Thanks for resolving it!